package.json

```diff
@@ -70,7 +71,7 @@
     "@types/node": "^13.11.1",
     "@types/semver": "^7.1.0",
     "@types/sinon": "^9.0.0",
-    "@types/vscode": "^1.52.0",
+    "@types/vscode": "^1.57.0",
```

Comment:
I needed to bump     "vscode": "^1.52.0" to the same version as this before it compiled for me.

Reply:
This will be bumped to 1.59 prior to release.